core/pta/widevine.c

```diff
@@ -0,0 +1,199 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2023, The ChromiumOS Authors
+ */
+
+#include <compiler.h>
+#include <initcall.h>
+#include <kernel/boot.h>
+#include <kernel/dt.h>
+#include <kernel/pseudo_ta.h>
+#include <libfdt.h>
+#include <malloc.h>
+#include <mm/tee_mm.h>
+#include <mm/tee_pager.h>
+#include <stdio.h>
+#include <string.h>
+#include <string_ext.h>
+#include <tee_api.h>
+#include <trace.h>
+#include <util.h>
+
+#define PTA_NAME "widevine.pta"
+
+#define GET_TPM_PUBKEY 0
+#define GET_WIDEVINE_PRIVKEY 1
+
+/* 721f4da9-da05-40d4-a1a3-8377c1e08b0a */
+#define WIDEVINE_PTA_UUID                                              \
```

Comment:
This UUID should be exposed from _lib/libutee/include/pta_widevine.h_ as done for other PTAs.